.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+    "root": true,
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "module"
+    },
+    "extends": ["plugin:prettier/recommended"],
+    "rules": {
+        "constructor-super": "error",
+        "dot-notation": "error",
+        "eqeqeq": "error",
+        "no-bitwise": "error",
+        "no-debugger": "error",
+        "no-eval": "error",
+        "no-extra-bind": "error",
+        "no-fallthrough": "error",
+        "no-new-wrappers": "error",
+        "no-plusplus": "error",
+        "no-restricted-globals": ["error", "event", "name", "length", "closed"],
+        "no-return-await": "error",
+        "no-sparse-arrays": "error",
+        "no-template-curly-in-string": "error",
+        "prefer-const": "warn",
+        "use-isnan": "error",
+        "unused-imports/no-unused-imports-ts": "error"
```

Comment:
No need for this rule, it is somehow covered by the `importNotUsedAsValues` in `tsconfig`